castle/src/main/java/io/castle/android/api/CastleAPIService.java

```diff
@@ -16,7 +16,8 @@
 
 public class CastleAPIService {
 
-    private static final String API_URL = "https://api.castle.io/v1/";
+    // public static final String API_URL = "https://api.castle.io/v1/";
+    public static final String API_URL = "http://192.168.64.18:3002/v1/";
```

Comment:
@aus this snuck into the release and AFAIK is currently in the released version.